app/brave_main_delegate.cc

```diff
@@ -38,7 +38,6 @@
 #include "components/omnibox/common/omnibox_features.h"
 #include "components/password_manager/core/common/password_manager_features.h"
 #include "components/safe_browsing/core/features.h"
-#include "components/security_state/core/features.h"
```

Comment:
is there a dep we can remove here?

Reply:
Doesn't look like we ever added any dependencies on `//components/security_state/core(:features)`